### PR TITLE
Adding "brotli-asgi" recipe

### PR DESCRIPTION
**Pull request by dcfidalgo**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dcfidalgo** [comment]
I confirm that I'm willing to be listed in the maintainer section.

**Comment by dcfidalgo** [comment]
@conda-forge-admin, please ping team 

This should be ready to review, thank you and have a nice day!

File: recipes/brotli-asgi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - brotli_asgi
+
+about:
+  home: "https://github.com/fullonic/brotli-asgi"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: "A compression AGSI middleware using brotli"
+  doc_url: 
+  dev_url:
```
Comment (ocefpaf) [inline recipes/brotli-asgi/meta.yaml:38]: Don't leave empty keys. You can remove those if you don't want to add the optional metadata there.

File: recipes/brotli-asgi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bcb815ba0c7b49a78df77bdede0d3fa62daaa429a326a1dcac293d69ef94858f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (ocefpaf) [inline recipes/brotli-asgi/meta.yaml:14]: This can probably be `noarch: python`.

Comment (dcfidalgo) [reply recipes/brotli-asgi/meta.yaml]: Yes, thanks for the tip!

File: recipes/brotli-asgi/meta.yaml (hunk 1 above)
Comment (dcfidalgo) [reply recipes/brotli-asgi/meta.yaml]: done

**Comment by dcfidalgo** [comment]
@ocefpaf Thanks a lot for the review, I applied your suggestions, as well as the general suggestions I received in [another PR](https://github.com/conda-forge/staged-recipes/pull/18123).

**Pull request merged**